docs/docs/installation.md

```diff
@@ -120,6 +120,10 @@ If you want to use it for a full application, there are two popular ways to get
 
 [Create React App](http://github.com/facebookincubator/create-react-app) is the best way to start building a new React single page application. It sets up your development environment so that you can use the latest JavaScript features, provides a nice developer experience, and optimizes your app for production.
 
+>**Note:**
+>
+>You'll need to have Node >= 6 on your machine.
```

Comment:
Maybe "CRA works best with Node.js version 6 or above" instead, since it's not actually required that node 6 be used?